src/servers/graphql-server.ts

```diff
@@ -108,7 +108,11 @@ export const startApolloServer = async ({
     },
   })
 
-  app.use(helmet({ contentSecurityPolicy: isDev ? false : undefined }))
+  app.use(
+    helmet({
+      contentSecurityPolicy: yamlConfig.helmet.disableContentPolicy ? false : undefined,
```

Comment:
@vindard has been doing so much work to remove all references to this global singleton. Can we not introduce one in the process?
Perhaps rebase off of https://github.com/GaloyMoney/galoy/pull/433 where he removes the other references to `yamlConfig` in this file?